repose-aggregator/components/services/opentracing-service/opentracing-service-api/src/test/scala/org/openrepose/core/services/opentracing/config/OpenTracingSchemaTest.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.core.services.opentracing.config
+
+import java.net.URL
+
+import org.openrepose.commons.test.ConfigurationTest
+import org.openrepose.core.service.opentracing.config.ObjectFactory
+import org.xml.sax.SAXParseException
+
+class OpenTracingSchemaTest extends ConfigurationTest {
+  override val schema: URL = getClass.getResource("/META-INF/schema/config/opentracing.xsd")
+  override val exampleConfig: URL = getClass.getResource("/META-INF/schema/examples/opentracing.cfg.xml")
+  override val jaxbContextPath: String = classOf[ObjectFactory].getPackage.getName
+
+  describe("schema validation") {
+    it("should successfully validate the config with http sender and sampling set to rate limited") {
```

Comment:
Why did you feel these cases needed testing? Typically we just do the example which you get for free by extending our test, and any assertions we put in the xsd.

Reply:
So this was originally added to get coverage on schema tests (similarly to AtomHopperSchemaTest).  Definitely not something that's required.